css/customs.css

```diff
@@ -0,0 +1,18 @@
+#playScreen {
+	margin-top: 30px;
+	;
+
+}
+
+.indiBranch {
+	stroke: rgba(139,69,19,0);
```

Comment:
[rgba](https://developer.mozilla.org/en/docs/Web/CSS/color_value#rgba()) defines the transparency/opacity of an element. In this case, it seems like your stroke is completely transparent... (value 0)